t/test-genome-DBs/multi/stable_ids/table.sql

```diff
@@ -7,17 +7,17 @@ CREATE TABLE `archive_id_lookup` (
   KEY `archive_id_db_type` (`archive_id`,`db_type`,`object_type`),
   KEY `archive_id_object_type` (`archive_id`,`object_type`),
   KEY `species_idx` (`species_id`)
-) ENGINE=MyISAM DEFAULT CHARSET=latin1 ;
+) ENGINE=MyISAM ;
```

Comment:
It must have been caused by the patching script. Do you need me to fix this for all tables in all the test_dbs table.sql?